### PR TITLE
Update ids to reflect new urls

### PR DESCRIPTION
**Pull request by iwootten**
### What is the context of this PR?

Update ids to match those in the url/page title in the spreadsheets: https://docs.google.com/spreadsheets/d/1BY1zRuVIL7ipsoVjg3V0fJ9vyaiWNH-AnsCC6O-LoKg/edit?pli=1#gid=0

### How to review

Check that the ids on gb/wls/nir have been updated as specified in the spreadsheet. (Good Luck!)

#### England

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-individual-urls/schemas/en/census_household_gb_eng.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-individual-urls/schemas/en/census_individual_gb_eng.json)

#### Wales

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-individual-urls/schemas/en/census_household_gb_wls.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-individual-urls/schemas/en/census_individual_gb_wls.json)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-individual-urls/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-individual-urls/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

File: source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet [hunk 1]
```diff
@@ -64,7 +64,7 @@ local additionalAnswerOption = [
 
 {
   type: 'Question',
-  id: 'past-usual-household-address',
+  id: 'location-one-year-ago',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet:67]: Should the question and answer ids also change?

File: source/jsonnet/england-wales/household/blocks/individual/individual_interstitial.jsonnet [hunk 2]
```diff
@@ -29,7 +29,7 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
           'main language',
           'general health',
           'unpaid care provided',
-          'qualifications',
+          'qualifications-introduction',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/household/blocks/individual/individual_interstitial.jsonnet:32]: This isn't an id.
```suggestion
          'qualifications',
```

File: source/jsonnet/england-wales/household/blocks/visitor/usual_household_address.jsonnet [hunk 3]
```diff
@@ -55,7 +55,7 @@ local question(additionalAnswerOptions=[]) = {
 
 {
   type: 'Question',
-  id: 'usual-household-address',
+  id: 'visitor-usual-address',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/household/blocks/visitor/usual_household_address.jsonnet:58]: Should the question and answer ids also change?

File: source/jsonnet/northern-ireland/household/blocks/individual/past_usual_household_address.jsonnet [hunk 4]
```diff
@@ -65,7 +65,7 @@ local additionalAnswerOption = [
 
 {
   type: 'Question',
-  id: 'past-usual-household-address',
+  id: 'location-one-year-ago',
```
Comment (ajmaddaford) [inline source/jsonnet/northern-ireland/household/blocks/individual/past_usual_household_address.jsonnet:68]: Should the question and answer ids also be updated?

File: source/jsonnet/northern-ireland/household/blocks/visitor/usual_household_address.jsonnet [hunk 5]
```diff
@@ -55,7 +55,7 @@ local question(additionalAnswerOptions=[]) = {
 
 {
   type: 'Question',
-  id: 'usual-household-address',
+  id: 'visitor-usual-address',
```
Comment (ajmaddaford) [inline source/jsonnet/northern-ireland/household/blocks/visitor/usual_household_address.jsonnet:58]: Should the question and answer id also be updated here?

File: source/jsonnet/northern-ireland/household/blocks/visitor/usual_household_address_other.jsonnet [hunk 6]
```diff
@@ -4,9 +4,9 @@ local rules = import 'rules.libsonnet';
 
 {
   type: 'Question',
-  id: 'usual-household-address-other',
+  id: 'visitor-usual-address-country',
   question: {
-    id: 'usual-household-address-other-question',
+    id: 'visitor-usual-address-other-question',
```
Comment (ajmaddaford) [inline source/jsonnet/northern-ireland/household/blocks/visitor/usual_household_address_other.jsonnet:9]: Should this be `visitor-usual-address-country-question` and `visitor-usual-address-country-answer`?

File: source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet (hunk 1 above)
Comment (iwootten) [reply source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet:67]: Probably, they didn't match in the first place meaning they were missed.

**Comment by iwootten** [comment]
> Is it worth now renaming the jsonnet files themselves where they match the old id names? I feel like it's a little bit confusing now it doesn't match although not sure as I appreciate that this could get messy 😐

Yeah, I'd wanted to do this, but missed it in the end.

**Comment by iwootten** [comment]
> Is it worth now renaming the jsonnet files themselves where they match the old id names? I feel like it's a little bit confusing now it doesn't match although not sure as I appreciate that this could get messy 😐

I've done it now, but it took part of my soul to do it.

**Pull request merged**